### PR TITLE
Added homepage

### PR DESCRIPTION
**Pull request by jameskokoska**
image
image
image

added home page
added footer wave
added footer to all pages

I'm not a fan of mustard yellow (for the registration button) @natapokie :( 

placeholders for missing home page components
https://github.com/UofT-Frosh-Orientation/orientation-website/issues/83
@lukewarmtemp please add to home page when complete
https://github.com/UofT-Frosh-Orientation/orientation-website/issues/84
@Freeassassin please add to home page when complete

### CODE REVIEW

**Comment by natapokie** [comment]
Yeah,,, i completely agree, the yellow is ugly 😭 sorry, sorry I'm gonna find another yellow... but it looks amazing so far !!

**Review by natapokie — APPROVED** [review]
👍👍👍

**Comment by natapokie** [comment]
okay, i'm gonna try and ask Kate again tomorrow, i'm sure there is something we can do about the purple !!

File: client/src/App.css [hunk 1]
```diff
@@ -14,13 +14,15 @@
...
   --green-success: #aad0a3;
   --red-error: #dd4343;
-  --text-link: #90caf9;
+  --text-link: #4471d9;
   --text-dark: #20002e;
   --text-light: #f3f3f3;
   --text-muted: #696969;
   --navbar-bg: #ebebeb;
   --navbar-underline: #e2e2e2;
   --page-bg: #f6f3fb; /* very light purple */
```
Comment (natapokie) [inline client/src/App.css:23]: also james, would it be possible to use this for the background colour of the page? 
or if you tried it already and it doesn't look good feel free to get rid of this!

**Review by natapokie — APPROVED** [review]

File: client/src/App.css (hunk 1 above)
Comment (jameskokoska) [reply client/src/App.css:23]: Yep! You can apply the background color to the body (and maybe the html tag if needed) in App.css

Comment (jameskokoska) [reply client/src/App.css:23]: you can submit a commit on this PR, or I can do it tomorrow

Comment (natapokie) [reply client/src/App.css:23]: yep! i can do that 👍

**Review by Cali0707 — APPROVED** [review]
Looks great!

**Pull request merged**